.banzaicloud/pipeline.yaml

```diff
@@ -37,13 +37,18 @@ pipeline:
     deployment:
       name: './spotguide-spark-1.0.0.tgz'
```

Comment:
Shouldn't the version number for the tgz change?

Reply:
spark version changed in requirements